luxtronik/__init__.py

```diff
@@ -58,9 +97,14 @@ def write(self):
             LOGGER.debug("Command %s", cmd)
             val = struct.unpack(">i", self._socket.recv(4))[0]
             LOGGER.debug("Value %s", val)
-        self._disconnect()
-        # flush queue after writing all values
+        # Flush queue after writing all values
         self.parameters.queue = {}
+        # Give the heatpump a short time to handle the value changes/calculations:
+        time.sleep(0.2)
```

Comment:
Ideally there should be no magic hardcoded values within the code itself. Rather this should be extracted into a "constant" and be named something like `WAIT_TIME_WRITE_PARAMETER`.